src/buddy/core/crypto.clj

```diff
@@ -42,7 +42,7 @@
 ;; --- Constants
 
 (def ^:no-doc
-  +block-cipher-engines+
+  ^BlockCipher +block-cipher-engines+
```

Comment:
I think that this type hint is wrong, because `+block-cipher-engines+` is a hash-map and not a `BlockCipher`

Reply:
Removed